web2/sdk/approver.ts

```diff
@@ -0,0 +1,36 @@
+import { multicall } from "@wagmi/core";
+import { IERC721_ABI } from "./constants";
+import { ERC721Contract } from "./contracts/erc721-contract";
+import { Item } from "./types";
+
+export class Approver {
+  private readonly flexiSwapAddress: Address;
+
+  constructor(flexiSwapAddress: Address) {
+    this.flexiSwapAddress = flexiSwapAddress;
+  }
```

Comment:
There is a syntax sugar in TS for this. Please use it for other occurrences as well
```suggestion
  constructor(private readonly flexiSwapAddress: Address) {}
```